gce.py

```diff
@@ -215,8 +215,12 @@ def node_to_dict(self, inst):
                 md[entry['key']] = entry['value']
 
         net = inst.extra['networkInterfaces'][0]['network'].split('/')[-1]
+        # If there is no public IP listed, set the public_ip field to a null.
         if inst.public_ips.__len__() == 0:
           inst.public_ips=[None]
+
+        # Use private IPs if we're told to, or we've got no choice.
+        if os.environ.has_key("ANSIBLE_USE_PRIVATE_SSH") or inst.public_ips==[None]:
```

Comment:
I wouldn't expect them to accept the environment variable upstream when the script already uses a config file.

So I think we should either do the quick and dirty thing to change the default in our forked version, or do the right thing and extend the config to something that could be PR'ed upstream.